qdrant/v1.2.x/points.md

```diff
@@ -299,12 +299,105 @@ client.upsert(
 )
 ```
 
+*Available since v1.2.0*
+
+Named vectors are optional. When uploading points, some vectors may be omitted.
+For example, you can upload one point with only the `image` vector and a second
+one with only the `text` vector.
+
+When uploading a point with an existing ID, the entire point is replaced.
+Unspecified vectors are set to null. In other words; when uploading such point,
+the existing point is deleted first, then it is inserted with just the specified
+vectors. To keep existing vectors unchanged and only update specified vectors,
+see [update vectors](#update-vectors).
+
 ## Modify points
 
-You can modify a point in two ways. The first is to modify the vector.
-Currently, you would need to re-upload the point to modify the vector.
+To change a point, you can modify its vectors or its payload. There are several
+ways to do this.
+
+### Update vectors
+
+*Available since v1.2.0*
+
+This method updates the specified vectors on the given points. Unspecified
+vectors are kept unchanged. All given points must exist.
+
+REST API ([Schema](https://qdrant.github.io/qdrant/redoc/index.html#operation/update_vectors)):
+
+```http
+PUT /collections/{collection_name}/points/vectors
+
+{
+    "points": [
+        {
+            "id": 1,
+            "vector": {
+                "image": [0.1, 0.2, 0.3, 0.4]
+            }
+        },
+        {
+            "id": 2,
+            "vector": {
+                "text": [0.9, 0.8, 0.7, 0.6, 0.5, 0.4, 0.3, 0.2]
+            }
+        }
+    ]
+}
+```
+
+```python
+client.update_vectors(
+    collection_name="{collection_name}",
+    points=[
+        models.PointStruct(
+            id=1,
+            vector={
+                "image": [0.1, 0.2, 0.3, 0.4],
+            },
+        ),
+        models.PointStruct(
+            id=2,
+            vector={
+                "text": [0.9, 0.8, 0.7, 0.6, 0.5, 0.4, 0.3, 0.2],
+            },
+        ),
+    ]
+)
+```
+
+To update points without keeping existing vectors intact, see [uploading
```

Comment:
```suggestion
To update points and replace all of its vectors, see [uploading
```

Reply:
I wrote it like this because I thought 'and replace all of its vectors' wouldn't make clear that it also nullifies unspecified vectors.

But this is definitely easier to understand, thanks for the suggestion.